sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/logging/HttpRequestLoggingInterceptor.java

```diff
@@ -19,17 +19,11 @@
 public class HttpRequestLoggingInterceptor implements HandlerInterceptor {
 
   private static final Logger logger = LoggerFactory.getLogger(HttpRequestLoggingInterceptor.class);
-
-  @Override
-  public boolean preHandle(
-      HttpServletRequest request, HttpServletResponse response, Object handler) {
-    return true;
-  }
+  private static final String TRACE_TOKEN_HEADER_NAME = "Trace-Token";
```

Comment:
Shouldn't we move all the code for trace tokens to `TraceTokenRequestInterceptor`?

Reply:
I would also like to have a single place where the `Trace-Token` header name is defined. I would agree that it is better located in a class covering `Trace` than this logging interceptor.